tests/standalone/Urllib3Using.py

```diff
@@ -0,0 +1,24 @@
+import urllib3
+
+# nuitka-skip-test-unless: urllib3
+
+# testing request
+http = urllib3.PoolManager()
+r = http.request('GET', 'http://httpbin.org/robots.txt')
+r = http.request('POST','http://httpbin.org/post',fields={'hello': 'world'})
+
+# testing response
+r.status
+r.data
+r.headers
+
+# testing JSON content
+import json
+r = http.request('GET', 'http://httpbin.org/ip')
+json.loads(r.data.decode('utf-8'))
+
+# testing binary content
+r = http.request('GET', 'http://httpbin.org/bytes/8')
```

Comment:
Using external resources kind of is a tough issue for the tests. They tend to be flaky, causing errors at non opportune times.

May I suggest to instead use standard library HTTP server package, and listen to a specific port, then making queries there.

Reply:
Also I would like to cover https:// protocol usage, as that is where for standalone the biggest challenges rest. You need the SSL libraries, and they often are having lots of dependencies that can be hidden. That way your test is likely to find something in some setups.

But also, Python may not be able to do SSL. I surely have seen versions that will give an exception. When that happens, just print it, much like many tests in `tests/basics` do, and print the exception, so we know we fail in the same way.

Reply:
I've tried using a standard library HTTP server, but the problem is that the script would be dedicated to running the server, and another script would be needed to make the queries. How can I do this without using multiprocessing or threading?

Reply:
I think threading ought to be OK for this test there. I don't think I want the test to be more than one file.